libass/ass.h

```diff
@@ -358,11 +362,14 @@ void ass_set_frame_size(ASS_Renderer *priv, int w, int h);
  * This is used to calculate the source aspect ratio and the blur scale.
  * The source image size can be reset to default by setting w and h to 0.
  * The value set with this function can influence the pixel aspect ratio used
- * for rendering.
+ * for rendering. The values must be from before anamorphic stretching.
```

Comment:
The meaning here isn't entirely clear; I'd say something like "The values must be the actual storage size of the video stream, without any anamorphic de-squeeze applied."

Reply:
Changed it to the suggested, thanks!